maintenance/image-options/imageset.md

```diff
@@ -167,12 +175,7 @@ EOF
 for x in "${imagelist[@]}"; do
   for y in ${images[*]}; do
     if [[ $x =~ $y: ]]; then
-      echo "Pulling $x"
-      docker pull $x -q > /dev/null
-      {% raw %}digests=$(docker inspect $x -f '{{range .RepoDigests}}{{printf "%s\n" .}}{{end}}'){% endraw %}
-      name=$(echo $x | sed -e 's|docker.io/||' -e 's|:.*$||')
-      imgdigest=$(echo -e $digests | grep $name)
-      digest=$(echo -e $imgdigest | sed -e 's|^.*@||')
+      digest=$(docker run --rm gcr.io/go-containerregistry/crane digest ${x})
```

Comment:
Is this a publicly available image?
I think we should include an image version. Should we copy the image to some place we control.

Reply:
Oops I meant to have the tag in there, will add it. Yes this image is publicly available on gcr: https://github.com/google/go-containerregistry/releases/tag/v0.7.0